install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -29,6 +29,19 @@ type WorkspaceConfig struct {
 		Limit            resource.Quantity `json:"limit"`
 		BurstLimit       resource.Quantity `json:"burstLimit"`
 	}
+
+	RegistryFacade struct {
```

Comment:
Why `registryFacade` is inside `workspace`?

Reply:
here `workspace` refers to the workspace team